unit-test.spec.js

```diff
@@ -483,5 +486,28 @@ describe('lookup', function () {
             expect(mime).have.property('format', 'xml');
             expect(mime).have.property('guessed');
         });
-    })
+    });
+    describe("Charset from headers", function () {
+        it("text/html; charset=Shift_JIS", function() {
+            var mime = mimeFormat.lookup('text/html; charset=Shift_JIS');
+            expect(mime).have.property('charset', 'shift_jis');
+        });
+
+        it("text/html; charset=ISO-8859-1", function() {
+            var mime = mimeFormat.lookup('text/html; charset=ISO-8859-1');
+            expect(mime).have.property('charset', 'iso-8859-1');
+        });
+    });
+
+    describe("Charset won't be available if it is not mentioned", function () {
```

Comment:
Need to change the text.. 
`Charset won't be available`

`if it is not mentioned` should be there in `it`